helpers/ViewHelper.php

```diff
@@ -70,6 +70,19 @@ public function dateago($datestr) {
         return \F3::get('lang_timestamp', $date->getTimestamp());
     }
 
+
+    /**
+     * Return ISO8601 formatted date
+     *
+     * @param sql date
+     * @return string
+     */
+    public static function date_iso8601($datestr) {
+        $date = new \DateTime($datestr);
+        return $date->format(\DateTime::ISO8601);
```

Comment:
`\DateTime::ISO8601` is not actually ISO 8601 compliant, `\DateTime::ATOM` should be used: https://secure.php.net/manual/en/class.datetime.php#datetime.constants.atom